arex-schedule-web-api/src/main/java/com/arextest/schedule/beans/ReplaySenderConfiguration.java

```diff
@@ -30,6 +30,8 @@
 public class ReplaySenderConfiguration {
   @Value("${replay.sender.extension.jarPath}")
   private String jarFilePath;
+  @Value("${replay.sender.extension.jarPath.remote:}")
```

Comment:
the attribute of jarPath is not well defined，can't it be distinguished by the protocol header?